build_simulators.sh

```diff
@@ -10,9 +10,9 @@ function test_build () {
     fi
 }
 
-test_build make ARCH=RV32 ocaml_emulator/riscv_ocaml_sim_RV32
-test_build make ARCH=RV64 ocaml_emulator/riscv_ocaml_sim_RV64
+#test_build make ARCH=RV32 ocaml_emulator/riscv_ocaml_sim_RV32
+#test_build make ARCH=RV64 ocaml_emulator/riscv_ocaml_sim_RV64
 
-test_build make ARCH=RV32 c_emulator/riscv_sim_RV32
-test_build make ARCH=RV64 c_emulator/riscv_sim_RV64
+test_build make ARCH=RV32 c_emulator/riscv_sim_RV32 -j24
+test_build make ARCH=RV64 c_emulator/riscv_sim_RV64 -j24
```

Comment:
Revert, Also if you have had issues with OCAML simulator, please let us know in https://github.com/riscv/sail-riscv/issues/138 comments